src/libraries/System.Security.Permissions/ref/System.Security.Permissions.Forwards.cs

```diff
@@ -17,3 +18,5 @@
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Security.PermissionSet))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Security.Permissions.PermissionState))]
 #endif
+
+#pragma warning restore SYSLIB0003
```

Comment:
Nit: I would not bother with restore at the end in these Forwards files (more than one place).